test/test_mps.py

```diff
@@ -2537,6 +2537,19 @@ def test_batch_norm_backward(self):
         # This used to crash, see https://github.com/pytorch/pytorch/issues/98602
         outputs.sum().backward()
 
+    # Regression test for https://github.com/pytorch/pytorch/issues/133520
+    def test_batch_norm_slices(self):
+        bn_cpu = nn.BatchNorm2d(100, affine=False, device='cpu')
+        bn_mps = nn.BatchNorm2d(100, affine=False, device='mps')
+
+        x_cpu = torch.randn(100, 100, 35, 45).to('cpu')
```

Comment:
Are these values fine, or should I consider lowering them to cover the minimal case? timeit results suggests a single loop is sub-millisecond.

Reply:
If it's sub-millisecond this is fine